nilearn/plotting/html_connectome.py

```diff
@@ -1,22 +1,60 @@
 import json
 
 import numpy as np
+from matplotlib import cm as mpl_cm
+from matplotlib import colors as mpl_colors
 from scipy import sparse
 
-from nilearn._utils import rename_parameters
 from .. import datasets
 from . import cm
 
 from .js_plotting_utils import (add_js_lib, mesh_to_plotly,
-                                encode, colorscale, get_html_template,
-                                to_color_strings)
+                                encode, colorscale, get_html_template)
 from nilearn.plotting.html_document import HTMLDocument
 
 
 class ConnectomeView(HTMLDocument):
     pass
 
 
+def _encode_coordinates(coords, prefix):
+    """
+    Transform a 2D-array of 3D data (x, y, z) into a dict of base64 values
+
+    Parameters
+    ----------
+    coords : ndarray, shape=(n_nodes, 3)
+        The coordinates of the nodes in MNI space.
+
+    prefix : str
+        Prefix for the key value in the returned dict. Schema is {prefix}{x|y|z}
+
+    Returns
+    -------
+    coordinates : dict
+        Dictionary containing base64 values for each axis
+    """
+    coordinates = {}
+
+    coords = np.asarray(coords, dtype='<f4')
+    marker_x, marker_y, marker_z = coords.T
+    for coord, cname in [(marker_x, "x"), (marker_y, "y"), (marker_z, "z")]:
+        coordinates["{}{}".format(prefix, cname)] = encode(
+            np.asarray(coord, dtype='<f4'))
+
+    return coordinates
+
+
+def _encode_markers_coordinates(coords):
```

Comment:
I am not sure these two functions add a lot of readability.
I would suggest replacing them with the call using the explicit arg name for the second argument, eg 
`_encode_coordinates(coords, prefix="_marker_")`
instead of `_encode_markers_coordinates(coords)` .

I am now thinking this module would maybe be slightly simpler if line and marker attributes were stored in separate (sub-)dictionaries inside the metadata, rather than using prefixes to implement "namespaces", but I don't think it's worth making this change now